ipymol/core.py

```diff
@@ -64,11 +65,10 @@ def display(self):
         fig = plt.figure(figsize=(20, 20))
         ax = fig.add_subplot(111)
         ax.axis('off')
-        fh = tempfile.NamedTemporaryFile()
-        self._server.do("cmd.png('%s')" % fh.name)
+        fh = tempfile.NamedTemporaryFile(suffix='.png')
+        self._server.do("png {};".format(fh.name))
```

Comment:
nit: if we're planning to use a Python 3 expression for this, I'd prefer to use an f-string:

```suggestion
        self._server.do(f'png {fh.name};')
```